.github/workflows/publish-nuget.yml

```diff
@@ -0,0 +1,103 @@
+name: Package and Publish (NuGet.org)
+on:
+  push:
+    branches:
+      - master
+    tags-ignore:
+      - '*' # don't loop
+
+jobs:
+  tag:
+    runs-on: ubuntu-latest
+    outputs:
+      result: ${{steps.s1.outputs.TAG_RESULT}}
+      version: ${{steps.s1.outputs.TAG_VERSION}}
+    steps:
+      - name: Determine Tag
+        id: s1
+        run: |
+          version=$(echo "${{github.event.head_commit.message}}" | perl -nle 'm/build:\s*([0-9]+.[0-9]+.[0-9]+)/; print $1');
+
+          if [ -z "$version" ];
+          then
+                  echo "No build version found";
+                  echo "TAG_RESULT=fail" >> $GITHUB_OUTPUT;
+          else
+                  export TAG_VERSION=$version;
+                  echo "TAG_VERSION=$TAG_VERSION" >> $GITHUB_OUTPUT;
+                  echo "TAG_RESULT=pass" >> $GITHUB_OUTPUT;
+          fi
+      - name: Create Tag
+        if: steps.s1.outputs.TAG_RESULT == 'pass'
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ github.token }}
+          script: |
+            github.git.createRef({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              ref: "refs/tags/v${{steps.s1.outputs.TAG_VERSION}}",
+              sha: context.sha
+            })
+  tag-lumina:
+    runs-on: ubuntu-latest
+    outputs:
+      result: ${{steps.s1.outputs.TAG_RESULT}}
+      version: ${{steps.s1.outputs.TAG_VERSION}}
+    steps:
+      - name: Determine Tag
+        id: s1
+        run: |
+          version=$(echo "${{github.event.head_commit.message}}" | perl -nle 'm/build-lumina:\s*([0-9]+.[0-9]+.[0-9]+)/; print $1');
+
+          if [ -z "$version" ];
+          then
+                  echo "No build version found";
+                  echo "TAG_RESULT=fail" >> $GITHUB_OUTPUT;
+          else
+                  export TAG_VERSION=$version;
+                  echo "TAG_VERSION=$TAG_VERSION" >> $GITHUB_OUTPUT;
+                  echo "TAG_RESULT=pass" >> $GITHUB_OUTPUT;
+          fi
+      - name: Create Tag
+        if: steps.s1.outputs.TAG_RESULT == 'pass'
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ github.token }}
+          script: |
+            github.git.createRef({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              ref: "refs/tags/lumina-v${{steps.s1.outputs.TAG_VERSION}}",
```

Comment:
This still says Lumina. Is that intentional?

Reply:
This is for the NetStone.GameData.Lumina. Should I seperate this in two actiooons for clarity?

Reply:
No, that's okay.